src/server/system_services/schemas/bucket_schema.js

```diff
@@ -111,6 +111,7 @@ module.exports = {
                             objectid: true // namespace resource id
                         },
                         path: { type: 'string' },
+                        force_md5_etag: { type: 'boolean' }, // enable md5 calculation per bucket
```

Comment:
based on our previous discussion, this configuration should be at the top level of the bucket. it shouldn't be under `write_resource`